CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+
+* Add support for Kubernetes 1.24 variants ([#2437])
+* Remove variant aws-k8s-1.19 ([#2316])
+* Increase the kube-api-server QPS from 5/10 to 10/20 ([#2436], thanks @tzneal)
+* Update eni-max-pods with new instance types ([#2416])
+* Add setting to change `kubelet`'s log level ([#2460], [#2470])
+
+## Platform Changes
+
+### AWS
+* Disable drivers for USB-attached network interfaces ([#2328])
+
+### Metal
+* Add driver support for Solarflare, Pensando, Myricom, Huawei, Emulex, Chelsio, Broadcom, AMD and Intel 10G+ network cards ([#2379])
+
+## Build Changes
+* Extend `external-files` to vendor go modules ([#2378], [#2403], [#2430])
+* Make `net_config` unit tests reusable across versions ([#2385])
+* Add diff-kernel-config to identify kernel config changes ([#2368])
+* Extended support for variants in buildsys ([#2339])
+* Clarify crossbeam license ([#2447])
+* Switch back to `simplelog` in `pubsys` ([#2454])
+* Honor `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` env variables when set ([#2425])
+* Use architecture specific json payloads in unit tests ([#2367], [#2363])
+* Add unified `check` target in `Makefile.toml` for review readiness ([#2384])
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+* Add `cargo make` tasks to perform migration testing in AWS K8s ([#2273])
+* Update Go dependencies of fist-party go projects ([#2424], [#2440], [#2450], [#2452], [#2456])
+* Update Rust dependencies ([#2458])
+* Update third-party packages ([#2397], [#2398], [#2464], [#2465], thanks @kschumy)
+* Update Bottlerocket SDK to 0.27.0 ([#2428])
+* Update AWS SDK Rust ([#2415], [#2476])
+* Update `testsys` dependencies ([#2392])
+* Fix `hotdog`'s spec URL to the correct upstream link ([#2326])
+* Fix clippy warnings and enable lints on pull requests ([#2337], [#2346], [#2443])
+* Format issue field in PR template ([#2314])
+
+## Documentation Changes
+* Update checksum for new `root.json` ([#2405])
+* Mention that boot settings are available in k8s-1.23 variants ([#2358])
+* Mention the need of AWS credentials in BUILDING.md and PUBLISHING-AWS.md ([#2334])
+* Add China to supported regions lists ([#2315])
+* Add community section to README.md ([#2305], [#2383])
+* Standardize `userdata.toml` as the filename used in different docs ([#2446])
+* Remove commit from image name in PROVISIONING-METAL.md ([#2312])
+* Add note to CONTRIBUTING.md that outlines filenames' casing ([#2306])
+* Fix typos in `Makefile.toml`, QUICKSTART-ECS.md, QUICKSTART-EKS.md, `netdog` and `prairiedog` ([#2318])
+* Fix casing for GitHub and VMware in CHANGELOG.md ([#2329])
+* Fix typo in test setup command ([#2477])
+* Fix TESTING.md link typo ([#2438])
+* Fix positional `fetch-license` argument ([#2457])
+
+[#2375]: https://github.com/bottlerocket-os/bottlerocket/pull/2375
+[#2204]: https://github.com/bottlerocket-os/bottlerocket/pull/2204
+[#2330]: https://github.com/bottlerocket-os/bottlerocket/pull/2330
+[#2445]: https://github.com/bottlerocket-os/bottlerocket/pull/2445
```

Comment:
nit: I would recommend sorting this list so it's easier to scan visually for duplicates.